turbo.json

```diff
@@ -10,8 +10,7 @@
       "outputs": ["dist/**"]
     },
     "download-keys": {
-      "inputs": ["./utils/download-proving-keys.ts"],
-      "outputs": ["bin/**"]
```

Comment:
similar to previous comment think we should keep an explicit `download-keys` script for CI purposes. I couldn't make some of the tests pass without it.